platform/ext/target/nrf5340/target_cfg.h

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2017-2019 Arm Limited
```

Comment:
Update copyright/add our own?

Reply:
Not really update - maybe add own, yes..

Reply:
Updated :)